<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>415</width>
    <height>229</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>9</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="gameBox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>189</width>
      <height>189</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QToolButton" name="box0">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>63</width>
       <height>63</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QToolButton" name="box3">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>63</y>
       <width>63</width>
       <height>63</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QToolButton" name="box6">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>126</y>
       <width>63</width>
       <height>63</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QToolButton" name="box1">
     <property name="geometry">
      <rect>
       <x>63</x>
       <y>0</y>
       <width>63</width>
       <height>63</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QToolButton" name="box7">
     <property name="geometry">
      <rect>
       <x>63</x>
       <y>126</y>
       <width>63</width>
       <height>63</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QToolButton" name="box4">
     <property name="geometry">
      <rect>
       <x>63</x>
       <y>63</y>
       <width>63</width>
       <height>63</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QToolButton" name="box2">
     <property name="geometry">
      <rect>
       <x>126</x>
       <y>0</y>
       <width>63</width>
       <height>63</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QToolButton" name="box5">
     <property name="geometry">
      <rect>
       <x>126</x>
       <y>63</y>
       <width>63</width>
       <height>63</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QToolButton" name="box8">
     <property name="geometry">
      <rect>
       <x>126</x>
       <y>126</y>
       <width>63</width>
       <height>63</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label0">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>63</width>
       <height>63</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label1">
     <property name="geometry">
      <rect>
       <x>63</x>
       <y>0</y>
       <width>63</width>
       <height>63</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label2">
     <property name="geometry">
      <rect>
       <x>126</x>
       <y>0</y>
       <width>63</width>
       <height>63</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label3">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>63</y>
       <width>63</width>
       <height>63</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label4">
     <property name="geometry">
      <rect>
       <x>63</x>
       <y>63</y>
       <width>63</width>
       <height>63</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label5">
     <property name="geometry">
      <rect>
       <x>126</x>
       <y>63</y>
       <width>63</width>
       <height>63</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label6">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>126</y>
       <width>63</width>
       <height>63</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label7">
     <property name="geometry">
      <rect>
       <x>63</x>
       <y>126</y>
       <width>63</width>
       <height>63</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label8">
     <property name="geometry">
      <rect>
       <x>126</x>
       <y>126</y>
       <width>63</width>
       <height>63</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <zorder>label8</zorder>
    <zorder>label7</zorder>
    <zorder>label6</zorder>
    <zorder>label5</zorder>
    <zorder>label4</zorder>
    <zorder>label3</zorder>
    <zorder>label2</zorder>
    <zorder>label1</zorder>
    <zorder>label0</zorder>
    <zorder>box8</zorder>
    <zorder>box7</zorder>
    <zorder>box6</zorder>
    <zorder>box3</zorder>
    <zorder>box4</zorder>
    <zorder>box5</zorder>
    <zorder>box2</zorder>
    <zorder>box1</zorder>
    <zorder>box0</zorder>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>246</x>
      <y>83</y>
      <width>126</width>
      <height>126</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QPushButton" name="newGame_button">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>126</width>
       <height>63</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="text">
      <string>New Game</string>
     </property>
    </widget>
    <widget class="QPushButton" name="settings_button">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>63</y>
       <width>126</width>
       <height>63</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Settings</string>
     </property>
    </widget>
   </widget>
   <widget class="QLineEdit" name="gameStatus_textBox">
    <property name="geometry">
     <rect>
      <x>229</x>
      <y>20</y>
      <width>166</width>
      <height>63</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>28</pointsize>
     </font>
    </property>
    <property name="inputMask">
     <string/>
    </property>
    <property name="maxLength">
     <number>16</number>
    </property>
    <property name="frame">
     <bool>false</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignCenter</set>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
    <property name="placeholderText">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
